packages/block-library/src/navigation-link/edit.js

```diff
@@ -129,6 +131,43 @@ function NavigationLinkEdit( {
 		selection.addRange( range );
 	}
 
+	function handleCreateEntity( type, entityTitle ) {
+		return saveEntityRecord( 'postType', type, {
+			title: entityTitle,
+			status: 'publish',
+		} ).then( ( entity ) => {
```

Comment:
Minor: You could turn the `handleCreateEntity` function into an `async` function and `await saveEntityRecord` to avoid an extra level of indentation with this `then` callback.

Reply:
Done.